### PR TITLE
Modernize Visual Studio AddIn csproj part 1

### PR DESCRIPTION
**Pull request by siegfriedpammer**
Cherry-picking changes from #2346 as requested by @sharwell.

@sharwell  are you OK with these changes?
@tom-englert fyi

### CODE REVIEW

File: ILSpy.AddIn/ILSpy.AddIn.csproj [hunk 1]
```diff
@@ -1,10 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project>
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
-
-  <PropertyGroup>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
-  </PropertyGroup>
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (sharwell) [inline ILSpy.AddIn/ILSpy.AddIn.csproj:2]:
```suggestion
<Project>
  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

**Review by sharwell — COMMENTED** [review]
I'm fine with this except for the one suggested change

File: ILSpy.AddIn/ILSpy.AddIn.csproj (hunk 1 above)
Comment (sharwell) [reply ILSpy.AddIn/ILSpy.AddIn.csproj:2]: 📝 The shorthand syntax (where these two lines are combined) implicitly adds two imports:

* It imports **Sdk.props** as the first item in the file
* It imports **Sdk.targets** as the last item in the file

Due to the requirement that **Microsoft.VsSDK.targets** be imported _after_ **Sdk.targets**, we cannot use the shorthand form and must instead explicitly import both **Sdk.props** and **Sdk.targets**.

Comment (siegfriedpammer) [reply ILSpy.AddIn/ILSpy.AddIn.csproj]: Fixed! OK?

**Review by sharwell — APPROVED** [review]

File: ILSpy.AddIn/ILSpy.AddIn.csproj (hunk 1 above)
Comment (sharwell) [reply ILSpy.AddIn/ILSpy.AddIn.csproj]: Yep!

Comment (tom-englert) [reply ILSpy.AddIn/ILSpy.AddIn.csproj]: > The shorthand syntax (where these two lines are combined) implicitly adds two imports:
> 
> * It imports **Sdk.props** as the first item in the file
> * It imports **Sdk.targets** as the last item in the file
> 
> Due to the requirement that **Microsoft.VsSDK.targets** be imported _after_ **Sdk.targets**, we cannot use the shorthand form and must instead explicitly import both **Sdk.props** and **Sdk.targets**.

=> This is exactly what the VSIX-SdkProjectAdapter package fixes.

**Pull request merged**